app/controllers/admin/merchants_controller.rb

```diff
@@ -5,5 +5,21 @@ def index
 
     def show
     end 
+
+
+    def toggle_status
+        user = User.find(toggle_params[:user_id])
+
+        user.switch_enabled
+
+        redirect_to admin_merchants_path
+    end 
+
+
+    private 
+
+    def toggle_params
+        params.permit(:enabled, :user_id)
```

Comment:
@nicovigil1, Strong params are used for user form inputs. 

This is admin functionality and is a button click which doesn't need strong params.

Reply:
el fin!